app/src/main/java/com/capstone/event_finder/network/EventApi.java

```diff
@@ -0,0 +1,114 @@
+package com.capstone.event_finder.network;
+
+import android.support.annotation.NonNull;
+
+import androidx.lifecycle.LiveData;
+
+import com.capstone.event_finder.models.Event;
+import com.google.gson.JsonArray;
+import com.google.gson.JsonObject;
+import com.parse.ParseUser;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+
+import retrofit2.Call;
+import retrofit2.Callback;
+import retrofit2.Response;
+
+public class EventApi {
+
+    private final EventRepository eventRepository;
+
+    public EventApi(EventRepository eventRepository) {
+        this.eventRepository = eventRepository;
```

Comment:
⚠️ We generally don't want to couple the API classes and Repository classes.  The `ViewModel` should be the only class that knows about both.

See below on how we can achieve this.